test/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    "extends": "tui",
+    "parserOptions": {
+        "ecmaVersion": 3
+    },
+    "env": {
+        "browser": true,
+        "jasmine": true,
+        "jquery": true,
+        "commonjs": true
+    },
+    "globals": {
+        "tui": true,
+        "loadFixtures": true
+    }
+};
```

Comment:
이 파일은 없어도 되겠네요~

https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy

When ESLint traverses into the tests/ directory, it will then use your-project/tests/.eslintrc in addition to your-project/.eslintrc. So your-project/tests/test.js is **linted based on the combination of the two**.

프로젝트 루트에 있는 `.eslintrc`랑 내용이 같아서 필요없어보이네요~
`max-nested-callbacks`처럼 test 파일들에만 적용할 룰들을 추가할 때만 필요하겠네용 ;ㅅ;

```
module.exports = {
    "rules": {
        "max-nested-callbacks": 0
    }
};
```

Reply:
테스트하다가 파일 삭제 안된채로 올라갔나보네요 ㅠㅠ 감사요~~ 저 파일은 필요없어요ㅎ